src/test/java/com/exasol/errorreporting/QuoterTest.java

```diff
@@ -20,14 +28,87 @@ static Stream<Arguments> getQuotingExamples() {
                 Arguments.of(List.of(1, "test"), "[1, 'test']"), //
                 Arguments.of(new CustomObject(), "<CustomObject>"), //
                 Arguments.of(null, "<null>"), //
-                Arguments.of("test", "'test'")//
+                Arguments.of("test", "'test'"), //
+                Arguments.of(new File(SEP + "foo" + SEP + "baz.txt"), "'"+ SEP + "foo" + SEP + "baz.txt'"), //
```

Comment:
This test is platform dependent.
```suggestion
                Arguments.of(new File("/foo/baz.txt"), "'/foo/baz.txt'"), //
                Arguments.of(new File("\\foo\\baz.txt"), "'\\foo\\baz.txt'"), //
```